akvo/templates/update_add.html

```diff
@@ -19,8 +19,8 @@
                 {% else %}
                 {% if edit_mode %}
                     <p class="small">
-                    {% blocktrans with update_time=update.time_gmt|date:"H:i T" expires_time=update.expires_at|date:"H:i T" %}
-                        You posted this update at {{update_time}}. You have until {{expires_time}} to save your edits.
+                    {% blocktrans with update_time=update.time_gmt|date:"H:i T" %}
```

Comment:
Let's add the date here as well. 

Previously you would not be able to see this on a different date (since we only allowed editing within 20 minutes), but now it's a bit weird to see "You posted this update at 14:29 GMT." when that was two weeks ago.